front50-s3/src/main/java/com/netflix/spinnaker/front50/model/S3Support.java

```diff
@@ -103,20 +104,30 @@ public boolean isHealthy() {
   }
 
   public T findById(String id) throws NotFoundException {
-    try {
-      S3Object s3Object = amazonS3.getObject(bucket, buildS3Key(id));
-      T item = deserialize(s3Object);
-      item.setLastModified(s3Object.getObjectMetadata().getLastModified().getTime());
-      return item;
-    } catch (IOException e) {
-      throw new IllegalStateException(e);
-    } catch (AmazonS3Exception e) {
-      if (e.getStatusCode() == 404) {
-        throw new NotFoundException(String.format("No item found with id of %s", id.toLowerCase()));
-      }
+    return new SimpleHystrixCommand<T>(
```

Comment:
If there's an issue retrieving from S3, attempt to serve a previously cached value